### PR TITLE
Dependabot for actions

### PR DESCRIPTION
**Pull request by hannesa2**
It updates you actions as well and sorry, I pushed to format file.

This is the only new content https://github.com/nextcloud/android/pull/8033/files#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28L3-L8
 
Ok, I make first a format commit ....

### CODE REVIEW

**Comment by hannesa2** [comment]
I split into code format commit and a change commit. So it's more easy to review

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 version: 2
 updates:
-    -   package-ecosystem: gradle
+    -   package-ecosystem: "github-actions"
```
Comment (AndyScherzinger) [inline .github/dependabot.yml:3]: package-ecosystem lines (see the one for gradle) differ in formatting: 3 spaces vs. no spaces

**Comment by AndyScherzinger** [comment]
Just a minor comment about formatting one line

File: .github/dependabot.yml (hunk 1 above)
Comment (hannesa2) [reply .github/dependabot.yml:3]: I only used this 
image

Comment (hannesa2) [reply .github/dependabot.yml:3]: Hmm, maybe after cherry-pick ?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,87 +1,91 @@
...
-    - ">= 1.2.16.a"
-  - dependency-name: pl.droidsonroids.gif:android-gif-drawable
-    versions:
-    - ">= 1.2.17.a"
-    - "< 1.2.18"
+    -   package-ecosystem: "github-actions"
+        directory: "/" # Location of package manifests
+        schedule:
+            interval: "weekly"
+    -package-ecosystem: gradle
```
Comment (hannesa2) [inline .github/dependabot.yml:7]: I guess this is the wrong line

Comment (AndyScherzinger) [reply .github/dependabot.yml]: Yes, I guess so. Don't know why it formatted 2 same lines differently, but might by due to the cherry pick.

**Comment by hannesa2** [comment]
Anyway I formatted all again. 

Btw, this format is ugly, the [standard format](https://github.com/hannesa2/LiveEdgeDetection/blob/master/.github/dependabot.yml) is more pretty, but tastes are different 😉

**Comment by AndyScherzinger** [comment]
> Btw, this format is ugly, the standard format is more pretty, but tastes are different 😉

I'd be fine with the changed yaml formatting, what do you think @tobiasKaminsky ?

**Comment by hannesa2** [comment]
Again, this is the only real change

```
-   package-ecosystem: "github-actions"
            directory: "/" # Location of package manifests
            schedule:
                interval: "weekly"
```

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**